Rubberduck.Resources/RubberduckUI.resx

```diff
@@ -171,8 +171,8 @@
   <data name="NameLabelText" xml:space="preserve">
     <value>Name:</value>
   </data>
-  <data name="OK" xml:space="preserve">
```

Comment:
Adding a new resource key is a good idea - renaming the OK key without adjusting the places referencing it is going to break all "Ok" captions though.

I would suggest keeping the OK resource key and adding a new one with the new caption, and then the settings dialog's ok button needs to use the new resource key.

Reply:
The resource key is referenced here:

https://github.com/rubberduck-vba/Rubberduck/blob/170191673789ec46cbd58519beaf443f42abeefa/Rubberduck.Core/UI/Settings/SettingsControl.xaml#L150

Reply:
Got it, will change that.

Reply:
Thanks!

Reply:
I believe this should do it.